docs/source/traitsui_user_manual/examples/array_editor.py

```diff
@@ -0,0 +1,42 @@
+# (C) Copyright 2004-2021 Enthought, Inc., Austin, TX
```

Comment:
This should almost definitely end up in `traitsui/examples/demo/StandardEditors`